docs/source/nlp/joint_intent_slot.rst

```diff
@@ -3,10 +3,253 @@
+| training_ds.prefix                        | string          | train                                                                            | A prefix for the training file names                                                                         |
++-------------------------------------------+-----------------+----------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------+
+| training_ds.num_workers                   | integer         | 2                                                                                | The number of worker threads for training                                                                    |
++-------------------------------------------+-----------------+----------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------+
+| training_ds.batch_size                    | integer         | 32                                                                               | The training data batch size                                                                                 |
++-------------------------------------------+-----------------+----------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------+
+| validation_ds.prefix                      | string          | dev                                                                              | A prefix for the validation file names                                                                       |
++-------------------------------------------+-----------------+----------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------+
+| validation_ds.num_workers                 | integer         | 2                                                                                | The number of worker threads for validation                                                                  |
++-------------------------------------------+-----------------+----------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------------------------+
+| validation_ds.batch_size                  | integer         | 32                                                                               | The validation data batch size                                                                               |
```

Comment:
please remove training/validation_ds params that are common for other models, probably only *.prefix should stay

Reply:
Corrected